implementations/micrometer-registry-otlp/src/test/java/io/micrometer/registry/otlp/OTelCollectorIntegrationTest.java

```diff
@@ -46,12 +48,25 @@ class OTelCollectorIntegrationTest {
 
     private static final String CONFIG_FILE_NAME = "collector-config.yml";
 
+    private static final DockerImageName COLLECTOR_IMAGE = DockerImageName
+        .parse("otel/opentelemetry-collector-contrib:" + getCollectorImageVersion());
+
     @Container
-    private final GenericContainer<?> container = new GenericContainer("otel/opentelemetry-collector-contrib")
+    private final GenericContainer<?> container = new GenericContainer(COLLECTOR_IMAGE)
         .withCommand("--config=/etc/" + CONFIG_FILE_NAME)
         .withClasspathResourceMapping(CONFIG_FILE_NAME, "/etc/" + CONFIG_FILE_NAME, READ_ONLY)
+        .waitingFor(Wait.forLogMessage(".*Everything is ready.*", 1))
```

Comment:
Maybe not needed, but it seemed like this was a better indication of being ready to receive traffic than the port being open (which I believe was the default wait strategy used before).